client/account-recovery/index.js

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
@mtias do you think I'd need to use the [auth controller](https://github.com/Automattic/wp-calypso/blob/master/client/auth/controller.js) or it'd be fine to have them separate and keep the current account recovery controller?